CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Contributing to Zephyr
```

Comment:
I think we should stick with ReST formatted documents rather than using markdown (GitHub supports a CONTRIBUTING.rst filename).